test/Libraries/RevitIntegrationTests/ElementTests.cs

```diff
@@ -37,6 +37,18 @@ public void CanDeleteElement()
             // query count node to verify 1 item deleted as a result of the wall deletion. 
             Assert.AreEqual(1, GetPreviewValue("ccd8a5ba37fd4b1297def564392ccf54"));
          }
-        
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanGetPinnedStatus()
+        {
+            string samplePath = Path.Combine(workingDirectory, @".\Element\deleteWallFromDocument.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
```

Comment:
is there a way to check there were no node `errors` or `warnings` raised after running the graph?
Maybe also check the return value of the node ?
We should do that as it's worthless otherwise - node could do nothing and we wouldn't know it.

Reply:
I think the line above `Assert.AreEqual(1, GetPreviewValue("ccd8a5ba37fd4b1297def564392ccf54"));` gets the output value of the node with that id 😉

Reply:
as far as I can see no other system test checks for `errors` or `warnings` - will ask on the next tech meeting if its possible.